test/PerformanceTests/ComponentTests/ODataWriter/ODataWriterFeedTests.cs

```diff
@@ -9,12 +9,12 @@ namespace Microsoft.OData.Performance
     using System;
     using System.IO;
     using System.Linq;
-    using global::Xunit;
     using Microsoft.OData;
     using Microsoft.OData.Edm;
     using Microsoft.Spatial;
-    using Microsoft.Xunit.Performance;
+    using BenchmarkDotNet.Attributes;
 
+    [MemoryDiagnoser]

```

Comment:
I think we should include class comments for test cases which give an overview of what is happening.

Reply:
Done